adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -93,13 +95,29 @@ - (void)initWithAppId:(nonnull NSString *)appId
   }
 
   _isInitializing = YES;
-  _isBannerPresenting = NO;
 
   // Disable refresh functionality for all banners
   [[VungleSDK sharedSDK] disableBannerRefresh];
 
+  // Set init options for priority placement
+  NSMutableDictionary *initOptions = [NSMutableDictionary dictionary];
+  if (delegate) {
+    NSString *priorityPlacementID = delegate.desiredPlacement;
+    [initOptions setObject:priorityPlacementID forKey:VungleSDKInitOptionKeyPriorityPlacementID];
+    _prioritizedPlacementID = [priorityPlacementID copy];
+
+    NSInteger priorityPlacementAdSize = 1;
+    if ([delegate respondsToSelector:@selector(bannerAdSize)]) {
+      GADAdSize adSize = [delegate bannerAdSize];
+      if (!GADAdSizeEqualToSize(adSize, kGADAdSizeMediumRectangle)) {
+        priorityPlacementAdSize = GADMAdapterVungleAdSizeForCGSize(adSize.size);
+      }
+    }
+    [initOptions setObject:[NSNumber numberWithInteger:priorityPlacementAdSize] forKey:VungleSDKInitOptionKeyPriorityPlacementAdSize];
```

Comment:
@YueVungle I see here you set init options to "unknown" for the size in the MREC case. is that ok?

Reply:
@ericleich Hi Eric, yes, only for Banner ad type (not include MREC) needs the Banner ad size.  Thanks 😊